imageroot/bin/module-dump-state

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+set -e
+
+mkdir -vp restore
+
+# test 
+if systemctl --user -q is-active mariadb-app; then
+    dbs=`podman exec mariadb-app mysql -BNre "show databases"` || exit 1
+    for db in $(echo $dbs)
```

Comment:
According to the container README, mysqldump allows to dump all databases in the same file. It could avoid iterating over each DB separately... What do you think?

See https://hub.docker.com/_/mariadb (section "Creating database dumps")

>     $ docker exec some-mariadb sh -c 'exec mariadb-dump --all-databases -uroot -p"$MARIADB_ROOT_PASSWORD"' > /some/path/on/your/host/all-databases.sql

Reply:
I thought about it, however the ns7 mariadb rpm does a relevant backup for each database, maybe if the whole backup fails, we could hope to restore individual sql dump.

I would prefer to go to individual dump but if you prefer I could do an unique dump (honestly at first I started for it)